packages/dnb-eufemia/src/shared/locales/da-DK.ts

```diff
@@ -0,0 +1,174 @@
+export default {
+  'da-DK': {
+    TextCounter: {
+      characterDown: '%count af %max tegn tilbage.',
+      characterUp: 'Du har brugt %count af %max tegn.',
+      characterExceeded: '%count tegn over grænsen på %max.',
+    },
+    TimelineItem: {
+      alt_label_completed: 'Udført',
+      alt_label_current: 'Nuværende',
+      alt_label_upcoming: 'Kommende',
+    },
+    Breadcrumb: {
+      navText: 'Sidehierarki',
+      goBackText: 'Tilbage',
+      homeText: 'Hjem',
+      backToText: 'Tilbage til...',
+    },
+    DateFormat: {
+      invalidDate: 'Ugyldig dato: {value}',
+    },
+    DatePicker: {
+      day: 'dag',
+      month: 'måned',
+      year: 'år',
+      start: 'fra',
+      end: 'til',
+      selectedDate: 'Valgt dato: %s',
+      selectedDateRange: 'Valgte datoer: %s',
+      selectedMonth: 'Valgt måned %s',
+      selectedYear: 'Valgt år %s',
+      nextMonth: 'Næste måned %s',
+      prevMonth: 'Forrige måned %s',
+      nextYear: 'Næste år %s',
+      prevYear: 'Forrige år %s',
+      openPickerText: 'Åbn datovælger',
+      maskOrder: 'dd/mm/yyyy',
+      maskPlaceholder: 'dd.mm.åååå', // have to be same setup as "mask" - but can be like: dd/mm/åååå
+      dateFormat: 'yyyy-MM-dd', // in v1 of date-fns we were more flexible in terms of the format
+      returnFormat: 'yyyy-MM-dd', // used in date-fns v1: YYYY-MM-DD
+      firstDay: 'monday', // used in DatePickerCalendar to set the first day of the week
+      submitButtonText: 'Ok',
+      cancelButtonText: 'Annuller',
+      resetButtonText: 'Nulstil',
+      placeholderCharacters: {
+        day: 'd',
+        month: 'm',
+        year: 'å',
+      },
+    },
+    Anchor: {
+      targetBlankTitle: 'Åbner et nyt vindue',
+    },
+    GlobalStatus: {
+      default_title: 'Der er opstået en fejl',
+      close_text: 'Luk',
+      status_anchor_text: 'Gå til %s',
+    },
+    GlobalError: {
+      404: {
+        title: 'Vi kan ikke finde den side, du leder efter …',
+        text: 'Er du sikker på, at du har skrevet den rigtige adresse? Eller har vi rodet med links?',
```

Comment:
```suggestion
        text: 'Er du sikker på, at du har skrevet den rigtige adresse?
```
```suggestion
        text: 'Er du sikker på, at du har skrevet den rigtige adresse?
        Hvis du har skrevet sidens adresse i adresselinjen, så kontroller, om adressen er indtastet korrekt.
```
```suggestion
        text: 'Siden du leder efter, findes ikke. Det kan være, fordi den er slettet eller flyttet.
        Hvis du har skrevet sidens adresse i adresselinjen, så kontroller, om adressen er indtastet korrekt.
```
+ en del flere formuleringer muligh her, men ikke "har vi rodet med links"